app/models/concerns/stacks_images.rb

```diff
@@ -0,0 +1,25 @@
+###
+#  Mixin to add methods for crafting Stacks image urls
+###
+module StacksImages
+  ##
+  # Generates a Stacks image url using given arguments
+  # @param [String] druid
+  # @param [String] image_id
+  # @param [Symbol] size
+  # @return [String]
+  def craft_image_url(druid, image_id, size)
```

Comment:
Yeah, probably.  Just curious, would the correct thing be here (for the 400px long-edge) to construct the druid + image id + `/full/!400,400/0/default.jpg` for this kind of thing?  Or is there another url pattern we should be using?